includes/blocks/course-theme/class-quiz-content.php

```diff
@@ -64,6 +64,7 @@ private static function render_questions_loop() {
 			<?php
 		}
 
+		remove_action( 'sensei_single_quiz_questions_after', array( 'Sensei_Quiz', 'the_quiz_pagination' ), 9 );
```

Comment:
Don't understand this. Is this something legacy that we have to stop doing? Why is it being removed "always" ?

Reply:
So the quiz pagination is automatically appended at the end of the post content. But in Learning Mode we need it in a separate place, in the footer tab (see below). The method `render_questions_loop` and the whole class `\Sensei\Blocks\Course_Theme\Quiz_Content` for that matter, is only rendered in the Learning Mode. Which means here we prevent the quiz pagination from being automatically appended into the post content and render it manually later where we need it to be.

I guess you're right. This line of code does not make sense without the context. I added a comment to clarify why it's there. cd4aebb2252c2b87bdc4deda9c695ef6c6cc58ed

Reply:
Thanks for the detailed explanation, @dadish 👍